.circleci/config.yml

```diff
@@ -31,6 +31,13 @@ references:
     run: |
       docker load -i /tmp/workspace/final_image.tar
 
+  install_pytest: &install_pytest
+    run: |
+      sudo apt-get update && sudo apt-get install -y python3-dev
+      curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
```

Comment:
Try the package `python3-pip` instead.  This doesn't pin the version of pip which kind of makes me wary of this.